cli/src/config.mjs

```diff
@@ -21,15 +21,16 @@
 'use strict';
 
 // IMPORTANT NOTES:
-// - app.defaultUserId here should match app.defaultUserId in the config.mjs file in the
+// - app.defaultUserId and app.defaultMerged here should match app.defaultUserId in the config.mjs file in the
 //   server directory/sub-repo and the value in the .mf-SAMPLE.config.json file also in
 //   this directory/sub-repo.
 
 const config = {
   app: {
     socketPort: 9998,
     httpPort: 3333,
-    defaultUserId: '12345'
+    defaultUserId: '12345',
+    defaultMerged: 'true'
```

Comment:
We should define the "default" merged state as true in the API definition. We don't have to add it as a configuration.

Reply:
Removed from config